utils/rjson.cc

```diff
@@ -193,6 +193,12 @@ struct guarded_yieldable_json_handler : public Handler {
 };
 
 void* internal::throwing_allocator::Malloc(size_t size) {
+    // For bypassing the address sanitizer failure in debug mode
```

Comment:
I think this comment doesn't quite describe what you're bypassing - that the sanitizer aborts on too-large single allocation instead of letting malloc fail normally.